website/src/components/react-native-plugins.js

```diff
@@ -0,0 +1,89 @@
+const React = require('react');
+import { useColorMode } from '@docusaurus/theme-common';
+const WidgetTabItems = () => {
+  const widgetsData = [
+    {
+      "tab": "Security Plugins",
+      "value": "securityPlugins",
+      "content": [
+        {
+          "lightIcon": "/learn/img/cards.svg",
+          "darkIcon": "/learn/img/cardsDark.svg",
+          "label": "Local Authentication",
+          "body": "Integrating Biometric Authentication such as facial recognition and fingerprint scanning for imporved security and user convenience. ",
+          "overview": "/learn/react-native/integrating-biometric-authentication",
+          "Plugin": "https://docs.expo.dev/versions/latest/sdk/local-authentication/"
+        },
+        {
+          "lightIcon": "/learn/img/cards.svg",
+          "darkIcon": "/learn/img/cardsDark.svg",
+          "label": "ScreenCapture",
+          "body": "Integrating Biometric Authentication such as facial recognition and fingerprint scanning for imporved security and user convenience. ",
+          "overview": "/learn/react-native/integrating-biometric-authentication",
+          "Plugin": "https://docs.expo.dev/versions/latest/sdk/local-authentication/"
+        },
+        {
+          "lightIcon": "/learn/img/cards.svg",
+          "darkIcon": "/learn/img/cardsDark.svg",
+          "label": "Device",
+          "body": "Integrating Biometric Authentication such as facial recognition and fingerprint scanning for imporved security and user convenience. ",
+          "overview": "/learn/react-native/detect-rooted-device",
+          "Plugin": "https://docs.expo.dev/versions/latest/sdk/local-authentication/"
```

Comment:
Why is the "plugin" link needed here? Can we just link one link

Reply:
Not necessarily. We will include only documentation links.